src/components/ViewSubSubHeader.js

```diff
@@ -0,0 +1,2 @@
+const ViewSubSubHeader = (props) => <header><h3>{props.children}</h3></header>
+export default ViewSubSubHeader
```

Comment:
Should this file be reverted, of the two? What was the point of this change?

Reply:
Yep, this was added by mistake. Removed!